### DISCUSSION TITLE
Import: I want to support multiple custom folders instead of the default originals

### DISCUSSION THREAD

**Discussion by softshield** [photoprism/photoprism#2820] [General]
Because there are a lot of ready-made pictures, we don't want to move them to originals.

I want to scan directly through a custom media library location. Just like the emby/jellyfin design.

**Comment by graciousgrey** [comment]
If you want to keep your folder structure you need to use indexing (not importing)

https://docs.photoprism.app/user-guide/library/

You don't need to move  your files. During setup you configure where your files are located (https://docs.photoprism.app/getting-started/docker-compose/#photoprismoriginals).

If you have multiple folders with files you can configure them as subfolders of originals as shown in our example docker-compose.yml files:
https://dl.photoprism.app/docker/

It could look like this:
```yaml
volumes:
      # "/host/folder:/photoprism/folder"                # Example
      - "/example/family:/photoprism/originals/family" # *Additional* media folders can be mounted like this`
      - "/yourpath/holiday:/photoprism/originals/holiday" # *Additional* media folders can be mounted like this`
```

**Comment by skrat** [comment]
@graciousgrey for those who do not use docker (aur, arch linux) and have external storage (exFAT) for both the originals and storage path, and would like to index certain (multiple) directories on that same storage device (ie. keep everything on the same ext. drive), what is your solution?

**Reply by graciousgrey** [reply]
You may use symbolic links for this https://docs.photoprism.app/known-issues/#symbolic-links

**Reply by skrat** [reply]
I checked that before I wrote my 1st comment. Most external drives are exfat FS, because it's the most portable at the moment. It doesn't support any kind of links. Refactoring the code, changing originals type from string to list<string>, or changing the iteration code, should not be that hard, what do you think? Guiding users to use symlinks is a hack anyways. It'd be great to also have UI for editing that list.
